Cooper/ClientApp/src.v2/app/services/authentification/authentification.service.ts

```diff
@@ -0,0 +1,122 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Router } from '@angular/router';
+import { AuthService, FacebookLoginProvider, GoogleLoginProvider } from 'ng-dynami-social-login';
+import { SocialNetwork} from '@enums';
+
+class LoginAttemptResponse {
+  token: string;
+}
+
+const registrationUrl = '/registration';
+const authUrl = '/auth/login';
+const socialProvider = {
+      'google': GoogleLoginProvider.PROVIDER_ID,
+      'facebook': FacebookLoginProvider.PROVIDER_ID
+    };
+
+@Injectable({
+  providedIn: 'root'
+})
+
+
+export class AuthentificationService {
+
+  constructor(private router: Router, private http: HttpClient, private socialAuthService: AuthService) {
+   }
+
+  public checkAuthentification(): void {
+    const token: string = localStorage.getItem('JwtCooper');
+    if (token) {
+      this.router.navigate(['/platform/home']);
```

Comment:
why function with name check changes state of program?

Reply:
Because I'm the one who calls himself ninja))